pytest_fixtures/api_fixtures.py

```diff
@@ -560,22 +553,17 @@ def module_env():
 
 
 @pytest.fixture(scope='module')
-def module_env_search(module_org, module_location, module_cv_with_puppet_module):
-    """Search for puppet environment according to the following criteria:
-    Content view from module_cv_with_puppet_module and chosen organization.
-
-    Returns the puppet environment with updated location.
+def module_env_search(module_org, module_location, module_import_puppet_module):
+    """Search for puppet environment created inside module_import_puppet_module.
+    Returns the puppet environment with updated organization and location.
```

Comment:
```suggestion
    """Search for puppet environment created from module_import_puppet_module fixture.
    
    Returns the puppet environment with updated organization and location.
```

Some wording and line between description and return statements!